### DISCUSSION TITLE
Cannot get streaming video after clicking on camera entity

### DISCUSSION THREAD

**Discussion by eruisi** [fuatakgun/eufy_security#799] [General]
I have T8221-Z smart doorbell with eufy homebase2 connected. I installed eufy security add-on and RTSP Simple Server Add-on then Eufy Security Integration. They used to to work flawlessly until eufy was hit by some data leaking rumors. After that I guess Eufy changed their data sharing protocol then I cannot get streaming video from doorbell when click on the entity.

By default, the camera entity show "Idle"
<img width="753" alt="Screenshot 2023-05-15 at 7 39 45 AM" src="https://github.com/fuatakgun/eufy_security/assets/26517770/f09c2f8a-461e-4431-b3f8-a385964168c4">

Only when you click on "Start P2P Streaming" button, the camera entity changes to "Streaming" then I can see video on HA and Homekit. and I have to click on "Stop P2P Streaming" button in the end to  save battery power. 
<img width="399" alt="Screenshot 2023-05-15 at 7 48 03 AM" src="https://github.com/fuatakgun/eufy_security/assets/26517770/e67e82c6-e73a-43a5-ac19-e936f9ca2a1f">

<img width="736" alt="Screenshot 2023-05-15 at 7 39 17 AM" src="https://github.com/fuatakgun/eufy_security/assets/26517770/04223d31-3f5f-44f6-9f3b-00bb003d1860">
 
Can we get the overall operation more streamlined? When I click on cameral entity in HA or Homekit, camera starts streaming automatically and stops streaming a few mins after closing video window in HA or Homekit.

**Comment by melsykin** [comment]
Same issue here, none of my cameras are able to stream except my doorbell for like 5 seconds if I start the P2P streaming. Also tried installing webrtc but it didnt help. Checking the rstp addon logs, I see a lot of "source of path <camera id?> has timed out"

**Comment by sven0833** [comment]
![Screenshot (96)](https://github.com/fuatakgun/eufy_security/assets/109326754/3ba47b5c-7afd-4ec5-b08b-25f1938ee5de)

i got same problem but got 1 of the camera working but the other 2 i cant get live feed from them, is there any fix i can do to fix this. seaching for a solution for 2 weeks now and nothing seems to work

**Comment by AseKarlsson** [comment]
Same issue here, did you solve it? 
Also some uncertainties how to setup the WebRTC?

**Reply by AseKarlsson** [reply]
Playing around with this some days now. Seems like everything else than Event Image and Video Stream is working as it should. (My unit is the (T8220-Z). Do I need to install WebRTC and if so, then how (Tried to install it from HACS and no errors, but still no Strem)?

All help appriciated!

Thanks,
Ase
